drivers/src/lms.rs

```diff
@@ -310,7 +372,10 @@ impl Lms {
         y: &[[U32<LittleEndian>; N]; P],
         message_digest: &HashValue<N>,
     ) -> CaliptraResult<HashValue<N>> {
-        let params = get_lmots_parameters(algo_type)?;
+        // wntz_mode: 1 for SHA256 with n=32, and 0 for SHA192 with n=24
```

Comment:
What part of the code is this comment describing?

Reply:
Removed.